packages/ui/react-ui-thread/src/Message/Message.tsx

```diff
@@ -41,14 +41,14 @@ export const MessageMeta = forwardRef<HTMLDivElement, MessageMetaProps>(
           className={mx('grid grid-cols-subgrid col-span-2', classNames)}
           ref={forwardedRef}
         >
-          <div role='none' className={mx('flex flex-col items-center gap-2', messageCell)}>
+          <div role='none' className={mx('flex flex-col items-center gap-2 plb-2 min-is-0')}>
```

Comment:
```suggestion
          <div role='none' className={mx('flex flex-col items-center gap-2 plb-2')}>
```
`min-is-0` should not be needed here, the `Avatar.Frame` already has an extrinsic size and should not need to shrink.

Reply:
ok thanks. note i bumped the padding on the icon so that the text more closely centers with it.